charts/qdrant/templates/statefulset.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 spec:
   replicas: {{ .Values.replicaCount }}
   podManagementPolicy: {{ .Values.podManagementPolicy }}
-  selector:
+    :
```

Comment:
Is that `selector` removal something you did on purpose? I do not understand if this was done on purpose.